AnomalyDetection/AnomalyDetection.py

```diff
@@ -28,8 +28,130 @@ def _partition_anomalies(windows, k):
     top_jump_positions = sorted(range(len(diffs)), key=lambda i: diffs[i], reverse=True)[0:k-1]
     return sorted(top_jump_positions) + [len(windows) - 1]
 
+def label_anomaly_windows(label):
+    """
+    :param label: input labels 1/0
+    :return: set of intervals of anomaly (start time, end time)
+    """
+    labels = label.values
+    timestamps = label.index
+    labeled_anomaly_window = []
+    for i in range(1, len(label)):
+        if labels[i] == 1 and labels[i-1] == 0:
+            start = timestamps[i]
+        elif labels[i-1] == 1 and labels[i] == 0:
+            end = timestamps[i-1]
+            labeled_anomaly_window.append((start, end))
+  
+    return labeled_anomaly_window
+
+def calculate_IOU(anomalies, label):
```

Comment:
It would be nice to annotate parameter types in the function definition. See this: https://docs.python.org/3/library/typing.html

For example: what is the type of label?